js/client.envirophat.ocf.js

```diff
@@ -0,0 +1,124 @@
+/** Copyright 2017 Vprime
+*
+* Licensed under the Apache License, Version 2.0 (the "License");
+* you may not use this file except in compliance with the License.
+* You may obtain a copy of the License at
+*
+*     http://www.apache.org/licenses/LICENSE-2.0
+*
+* Unless required by applicable law or agreed to in writing, software
+* distributed under the License is distributed on an "AS IS" BASIS,
+* WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+* See the License for the specific language governing permissions and
+* limitations under the License.*/
+
+/** @author Rami Alshafi <ralshafi@vprime.com> */
+
+var OCProcessInterval;
+var valid_input = ["0", "1", "2", "3", "4", "5", "6"];
+var iotivity = require( "iotivity-node/lowlevel" );

```

Comment:
@alshafi Thanks for adding samples. How about using high level OCF JS API https://github.com/01org/iot-js-api/tree/master/ocf instead of lowlevel API?

Reply:
I was familiar with the low level OCF JS API but possibly in the future whenever I get more bandwidth I will write another set of sample applications using the high level APIs